core/vm/jump_table.go

```diff
@@ -108,6 +108,7 @@ func newCancunInstructionSet() JumpTable {
 	enable1153(&instructionSet) // EIP-1153 "Transient Storage"
 	enable5656(&instructionSet) // EIP-5656 (MCOPY opcode)
 	enable6780(&instructionSet) // EIP-6780 SELFDESTRUCT only in same transaction
+	enable7702(&instructionSet)
```

Comment:
!??!?!?!?!?

Reply:
Wow nice catch. Must've added this early on and forgot to move to it's own definition. Fixed in [857641d](https://github.com/ethereum/go-ethereum/pull/30078/commits/857641d8f528fc90b1a98c268d525407035c0d03).